### PR TITLE
New package: mininet

### PR DESCRIPTION
**Pull request by renatoaguiar**

### CODE REVIEW

File: srcpkgs/mininet/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+pycompile_module="mininet"
+hostmakedepends="help2man python-setuptools"
+depends="bash inetutils-telnet iperf iputils libcgroup-utils net-tools openvswitch python socat"
+short_desc="Emulator for rapid prototyping of Software Defined Networks"
+maintainer="Renato Aguiar <contact@renatoaguiar.org>"
+license="custom"
+homepage="http://mininet.org/"
+distfiles="https://github.com/mininet/${pkgname}/archive/${version}.tar.gz"
+checksum=e4c20a6ab4ddfe4d5cad06ba25742507967df67a845dbe2e90d627af7cd21705
+
```
Comment (Vaelatern) [inline srcpkgs/mininet/template]: `noarch=yes`

Comment (renatoaguiar) [inline srcpkgs/mininet/template]: It also includes a binary (mnexec) which is arch dependent. This binary is compiled on post_build and installed on post_install.

File: srcpkgs/mininet/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+pkgname=mininet
+version=2.2.1
+revision=1
+build_style=python-module
+pycompile_module="mininet"
+hostmakedepends="help2man python-setuptools"
+depends="bash inetutils-telnet iperf iputils libcgroup-utils net-tools openvswitch python socat"
+short_desc="Emulator for rapid prototyping of Software Defined Networks"
+maintainer="Renato Aguiar <contact@renatoaguiar.org>"
+license="custom"
```
Comment (Vaelatern) [inline srcpkgs/mininet/template]: BSD

**Comment by Vaelatern** [comment]
Disregard my comment about BSD. Was confused.

File: srcpkgs/mininet/template [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+post_build() {
+	make mnexec
+	make man
+}
+
+post_install() {
+	vbin mnexec
+	vman mn.1
+	vman mnexec.1
+	vlicense LICENSE
```
Comment (Vaelatern) [inline srcpkgs/mininet/template]: Sorry, was wrong, isn't a BSD license, was confused.

**Comment by renatoaguiar** [comment]
In the website it says BSD, but the text in the LICENSE file  is not exactly the same. I'm going to change it back to custom.

"Mininet is actively developed and supported, and is released under a permissive BSD Open Source license." (http://mininet.org)

**Comment by Vaelatern** [comment]
I do suspect the `cc` in the Makefile should be patched to `$(CC)` but since the build is succeeding I'm not sure it matters.

File: srcpkgs/mininet/template [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# Template file for 'mininet'
+pkgname=mininet
+version=2.2.1
+revision=1
+build_style=python-module
+pycompile_module="mininet"
+hostmakedepends="help2man python-setuptools"
+depends="bash inetutils-telnet iperf iputils libcgroup-utils net-tools openvswitch python socat"
```
Comment (Vaelatern) [inline srcpkgs/mininet/template]: `python` is an implied dependency in this case.

**Comment by Vaelatern** [comment]
Last comment, the commit should be `New package: mininet-2.2.1`

**Pull request merged**